rewrites.js

```diff
@@ -15,7 +15,7 @@ exports.REWRITES = [
     destination: '/staticPage',
   },
   {
-    source: '/foundation/apply/:step(intro|fees|form|success)?',
+    source: '/foundation/apply/new/:step(intro|fees|form|success)?',
```

Comment:
Can we avoid introducing `new` here and simply send to a `step` (`intro` or `form`) when required?

Reply:
Yes we can avoid `new` here